src/features/tags/components/TagItem.js

```diff
@@ -1,23 +1,21 @@
 import React, { Component } from "react";
-import { NavLink } from "react-router-dom";
 import { connect } from "react-redux";
 import { getTagById } from "../selectors";
+import { Link } from "@mui/material";
 
 class TagItem extends Component {
   render() {
     const { tag, key } = this.props;
 
     return (
       <li className="menu__list-item" name={tag.id} key={key}>
-        <NavLink
-          exact
-          to={`/tag/${tag.id}`}
-          activeClassName="menu__link menu__link--active active"
-          className="menu__link"
+        <Link 
+          href={`/tag/${tag.id}`}
+          name={tag.id}
+          key={key}
         >
           #{tag.tag_name}
-          {tag.count > 1 && <span>{tag.count}</span>}
```

Comment:
This code should not be deleted. It displays how many articles have been assigned a specific tag